commit c89be18b5eda0dfaff193b65f3bfc648a8501bd0
Author: Giloo <33936193+GillesDuvert@users.noreply.github.com>
Date:   Sat Dec 13 16:59:19 2025 +0100

    Remove spurious unneeded include
    
    Removed unnecessary wxWidgets graphics header include that was introduced by the IDE, NetBeans.

diff --git a/src/gdlwxstream.cpp b/src/gdlwxstream.cpp
index d417eee6..de803aa5 100644
--- a/src/gdlwxstream.cpp
+++ b/src/gdlwxstream.cpp
@@ -15,8 +15,6 @@
  *                                                                         *
  ***************************************************************************/
 
-#include <wx-3.0/wx/graphics.h>
-
 #include "includefirst.hpp"
 
 #ifdef HAVE_LIBWXWIDGETS
